assets/js/components/SapSystemDetails/GenericSystemDetails.jsx

```diff
@@ -28,15 +31,41 @@ const getUniqueHosts = (hosts) =>
       .values()
   );
 
-export function GenericSystemDetails({ title, type, system }) {
+export function GenericSystemDetails({
+  title,
+  type,
+  system,
+  onInstanceCleanUp,
+}) {
   if (!system) {
     return <div>Not Found</div>;
   }
+  const [cleanUpModalOpen, setCleanUpModalOpen] = useState(false);
+  const [instanceToDeregister, setInstanceToDeregister] = useState(undefined);
+  const [instanceType, setInstanceType] = useState(undefined);
```

Comment:
We don't need this `instanceType`, right?
The `type` is already coming as argument to the view